ci/si_install_deps.sh

```diff
@@ -22,5 +22,10 @@ if ! command -v amm >/dev/null 2>&1; then
     echo "Ammonite successfully installed"
 fi
 
+# Ensure timeout is available.
+if [ "$PLATFORM" == 'Darwin' ]; then
```

Comment:
One has to have `brew install coreutils` but ok.

Reply:
I'll add that.